### DISCUSSION TITLE
Scrobbling

### DISCUSSION THREAD

**Discussion by calvinthefreak** [audioling/audioling#19] [Ideas]
Generall Playing Info reported to the Server.
I like to scrobble my history from Navidrome to Last.fm and Listenbrains.

**Comment by jeffvli** [comment]
This will be implemented as it's a core Subsonic feature.

**Comment by GlassedSilver** [comment]
Hijacking this since the proposed scope is planned anyhow.

Can you consider adding client-side scrobbling to Last.fm?

I don't scrobble server-side to Last.fm, because I like being able to live scrobble regardless of connection-status.
(and also because it works regardless of player used on Android through Panoscrobbler, which isn't a thing on Windows :s)

Also, plugins aren't user-aware, so you scrobble for everyone or nobody server-side. (Jellyfin user here)
